5-Heroku/server/db/getAllItemsFromDB.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose')
+const itemModel = require('../models/item')
+
+async function getAllItemsFromDB(userID) {
```

Comment:
nice separation for files. however for simple CRUD actions i think it's a bit overkill. I would've put all CRUD operation in a `TodoDAO.js` 

DAO = data access object (read about it)

Reply:
* i meant the to: 
uinte the 3 lines. u can "choose" the [action](https://github.com/haimre/karate-kid-kickstart/pull/classes.strike) based on logic
const action = 'remove';
titleElement.classListaction

Reply:
no, I meant to unite the files `getAllITems.js` `getItem.js` ...

Reply:
did u unite the files? I can still see separate files